### ISSUE TITLE
[Proposal] Add issue templates

### ISSUE THREAD

**Issue by duanwilliam** [acmucsd/main-website#245]
cc @trulyronak @steets250 

We should add issue and/or PR templates to make issues and PRs easier to diagnose, understand, etc. This would also help with a greater shift toward encouraging any member to contribute to ACM's development projects.

We may want to create a standardized set of issue templates across repos of similar domains (e.g. main website, membership portal UI, each community's own websites, etc share issue templates since they're all repos for a React frontend).

**Comment by trulyronak** [comment]
Agree, I really want a set of standardized practices across *all* repos

- Linting
- Using yarn over non
- Tabs LOL 
- Pr template 
- Issue template
- Contributing guide

**Comment by StormFireFox1** [comment]
We already fortunately have a repo I made a while back for this at https://github.com/acmucsd/contribution-templates. I have some basic setup for web frontend, we can use that for other projects as well.

Push comes to shove, we can make template repos that abide our standards, which I think is more useful for all the other points (linting, etc.)
